







<!doctype html>
<html>
  
<!-- Mirrored from sources.debian.net/src/dds/2.5.2+ddd105-1/debian/patches/sanitize-trydds.py by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 24 Jul 2014 06:08:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    
  
    <meta charset="UTF-8">
    <link rel="stylesheet" type="text/css"
          href="../../../../../static/css/debian.css" />
    <link rel="stylesheet" type="text/css"
          href="../../../../../static/css/base.css" />
    <link rel="shortcut icon"
          href="http://sources.debian.net/static/favicon.ico" />
    <title>File: sanitize-trydds.py | Debian sources</title>
    

  <link rel="stylesheet"
        href="../../../../../javascript/highlight/styles/googlecode.css">
  <script src="../../../../../javascript/highlight/highlight.pack.js"></script>
  <link rel="stylesheet" type="text/css"
        href="../../../../../static/css/source_file.css" />


  </head>
  <body>
    <header id="header">
      <div id="upperheader">
        <div id="logo">
          <a href="http://debian.org/" title="Debian Home"><img src="../../../../../static/img/debian-50.png" alt="Debian"></a>
        </div> <!-- end logo -->
        <p class="section"><a href="http://sources.debian.net/">Debsources</a></p>
	<div id="searchbox">
	    <form action="http://sources.debian.net/search/" name="searchform"
        method="post" style="display: inline;">
      <input id="query-1" name="query" placeholder="package name" type="text" value="">
    
    
    <input type="submit" value="Search package" />
  </form>
	    <form name="codesearch" method="get"
		  action="http://codesearch.debian.net/search">
	      <input name="q" value="package:dds "
		     type="text" />
	      <input type="submit" value="Search code" />
	    </form>
	</div>   <!-- end sitetools -->
      </div> <!-- end upperheader -->
      <!--UdmComment-->
      <nav id="navbar">
        <p class="hidecss"><a href="#content">Skip Quicknav</a></p>
        <ul>
          <li><a href="http://sources.debian.net/">Home</a></li>
          <li><a href="http://sources.debian.net/advancedsearch/">Search</a></li>
          <li><a href="http://sources.debian.net/doc/">Documentation</a></li>
          <li><a href="http://sources.debian.net/stats/">Stats</a></li>
          <li><a href="http://sources.debian.net/about/">About</a></li>
        </ul>
      </nav> <!-- end navbar -->
      <p id="breadcrumbs">source / <a href="http://sources.debian.net/src/dds">dds</a> / <a href="../../../2.5.html">2.5.2+ddd105-1</a> / <a href="../../debian.html">debian</a> / <a href="../patches.html">patches</a> / sanitize-trydds.py</p>
    </header> <!-- end header -->
    
    <div id="content">
      


<h2>File: sanitize-trydds.py</h2>



<script type="text/javascript">
    function toggle(id)
    {
    var elem = document.getElementById(id);
    if(elem.style.display == "none")
      elem.style.display = "block";
    else
      elem.style.display = "none";
    }
  </script>


<div id="pkginfobox" class="pkginfobox_fixed">

  
  <span onclick="toggle('infobox_content')">package info
    <small>(click to toggle)</small></span>
  
  <div id="infobox_content">
    
    <em>dds 2.5.2+ddd105-1</em>
    
    <ul>
      <li>links:
	<a href="http://packages.qa.debian.org/dds"><abbr title="Debian Package Tracking
							   System">PTS</abbr></a>,
	<a href="http://svn.df7cb.de/bridge/dds/trunk"><abbr title="Version Control System">VCS</abbr></a>
	</li>
      <li>area: main</li>
      <li>in suites: jessie, sid</li>
      
        <li>size: 1,552 kB</li>
      
      
        <li><abbr title="source lines of code">sloc</abbr>:
	  
            
              cpp: 8,945; 
            
              ansic: 281; 
            
              python: 89; 
            
              makefile: 87
            
	  
        </li>
      
    </ul>
  </div>
</div>


<table id="file_metadata">
  <tr>
    <td>
    file content (48 lines)
    | permissions: rw-r--r--
    </td>
    <td style="text-align: right;">
    <a href="../patches.html">parent folder</a>
    | <a href="http://sources.debian.net/data/main/d/dds/2.5.2+ddd105-1/debian/patches/sanitize-trydds.py">download</a>
    
    | <a href="http://sources.debian.net/sha256/?checksum=4730f4d442031c28b3da051a8978b78e082f96f036ee28e65234f26af3a2bc1c&amp;page=1">
	  duplicates (4)</a>
    
    </td>
  </tr>
</table>

<table id="codetable">
  <tr>
    <td>
      <pre id="sourceslinenumbers"><a id="L1" href="#L1">1</a><br /><a id="L2" href="#L2">2</a><br /><a id="L3" href="#L3">3</a><br /><a id="L4" href="#L4">4</a><br /><a id="L5" href="#L5">5</a><br /><a id="L6" href="#L6">6</a><br /><a id="L7" href="#L7">7</a><br /><a id="L8" href="#L8">8</a><br /><a id="L9" href="#L9">9</a><br /><a id="L10" href="#L10">10</a><br /><a id="L11" href="#L11">11</a><br /><a id="L12" href="#L12">12</a><br /><a id="L13" href="#L13">13</a><br /><a id="L14" href="#L14">14</a><br /><a id="L15" href="#L15">15</a><br /><a id="L16" href="#L16">16</a><br /><a id="L17" href="#L17">17</a><br /><a id="L18" href="#L18">18</a><br /><a id="L19" href="#L19">19</a><br /><a id="L20" href="#L20">20</a><br /><a id="L21" href="#L21">21</a><br /><a id="L22" href="#L22">22</a><br /><a id="L23" href="#L23">23</a><br /><a id="L24" href="#L24">24</a><br /><a id="L25" href="#L25">25</a><br /><a id="L26" href="#L26">26</a><br /><a id="L27" href="#L27">27</a><br /><a id="L28" href="#L28">28</a><br /><a id="L29" href="#L29">29</a><br /><a id="L30" href="#L30">30</a><br /><a id="L31" href="#L31">31</a><br /><a id="L32" href="#L32">32</a><br /><a id="L33" href="#L33">33</a><br /><a id="L34" href="#L34">34</a><br /><a id="L35" href="#L35">35</a><br /><a id="L36" href="#L36">36</a><br /><a id="L37" href="#L37">37</a><br /><a id="L38" href="#L38">38</a><br /><a id="L39" href="#L39">39</a><br /><a id="L40" href="#L40">40</a><br /><a id="L41" href="#L41">41</a><br /><a id="L42" href="#L42">42</a><br /><a id="L43" href="#L43">43</a><br /><a id="L44" href="#L44">44</a><br /><a id="L45" href="#L45">45</a><br /><a id="L46" href="#L46">46</a><br /><a id="L47" href="#L47">47</a><br /><a id="L48" href="#L48">48</a><br /></pre>
    </td>
    <td>
      <pre><code id="sourcecode" class="python">Index: trunk/python/trydds.py
===================================================================
--- trunk.orig/python/trydds.py	2007-01-19 01:02:23.212921160 +0100
+++ trunk/python/trydds.py	2007-01-19 01:05:49.711528600 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/python
+#!/usr/bin/python
 
 import sys
 import pydds
@@ -10,10 +10,10 @@ timer = time.time
 def dod(ds):
     t = timer()
     d = timer()
-    # deal = [h.split(&#39; &#39;) for h in ds.split(&#39;|&#39;)]
-    deal = [h.split(&#39;.&#39;) for h in ds.split()[1::2]]
+    deal = [h.split(&#39; &#39;) for h in ds.split(&#39;|&#39;)]
+    #deal = [h.split(&#39;.&#39;) for h in ds.split()[1::2]]
     d = timer() - d
-    print &#39;D&#39;, 8*&#39;.&#39;,&#39;|&#39;.join(&#39; &#39;.join(s) for s in deal)
+    #print &#39;D&#39;, 8*&#39;.&#39;,&#39;|&#39;.join(&#39; &#39;.join(s) for s in deal)
     # sys.stdout.flush()
     a = timer()
     dd = pydds.deal(deal)
@@ -26,17 +26,17 @@ def dod(ds):
     b = timer() - b
     t = timer() - t
     print &#39;T%d&#39; % s[-1][0],
-    print &#39;%.2f %.2f %.2f %.2f&#39; % (d, a, b, t)
+    #print &#39;%.2f %.2f %.2f %.2f&#39; % (d, a, b, t)
     # print &#39;|&#39;.join(&#39; &#39;.join(s) for s in deal),
     print s
     return t
 
 
-# ds = &#39;AQ9 AQ9 8643 KQ8|T82 T6542 AKT A7|K653 K83 Q92 543|J74 J7 J75 JT962&#39;
-ds=&#39;n AQ3.T653.KQ4.AJ5 e KJT4.AKQ8.JT86.8 s 862.974.A73.KQ72 w 975.J2.952.T9643&#39;
-print ds
-dod(ds)
-sys.exit()
+#ds = &#39;AQ9 AQ9 8643 KQ8|T82 T6542 AKT A7|K653 K83 Q92 543|J74 J7 J75 JT962&#39;
+##ds=&#39;n AQ3.T653.KQ4.AJ5 e KJT4.AKQ8.JT86.8 s 862.974.A73.KQ72 w 975.J2.952.T9643&#39;
+#print ds
+#dod(ds)
+#sys.exit()
 
 def main():
     handsfile = open(&#39;t1&#39;)
</code></pre>
    </td>
    
  </tr>
</table>

<script type="text/javascript">
  hljs.highlightBlock(document.getElementById('sourcecode'))
</script>




    </div>
    <footer id="footer">
      

<p style="margin: 0 0 0 0; line-height: 1em;">
  Browse by prefix: &ensp;
  
    <a href="http://sources.debian.net/prefix/0/">0</a>
    <a href="http://sources.debian.net/prefix/2/">2</a>
    <a href="http://sources.debian.net/prefix/3/">3</a>
    <a href="http://sources.debian.net/prefix/4/">4</a>
    <a href="http://sources.debian.net/prefix/6/">6</a>
    <a href="http://sources.debian.net/prefix/7/">7</a>
    <a href="http://sources.debian.net/prefix/8/">8</a>
    <a href="http://sources.debian.net/prefix/9/">9</a>
    <a href="http://sources.debian.net/prefix/W/">W</a>
    <a href="http://sources.debian.net/prefix/a/">a</a>
    <a href="http://sources.debian.net/prefix/b/">b</a>
    <a href="http://sources.debian.net/prefix/c/">c</a>
    <a href="http://sources.debian.net/prefix/d/">d</a>
    <a href="http://sources.debian.net/prefix/e/">e</a>
    <a href="http://sources.debian.net/prefix/f/">f</a>
    <a href="http://sources.debian.net/prefix/g/">g</a>
    <a href="http://sources.debian.net/prefix/h/">h</a>
    <a href="http://sources.debian.net/prefix/i/">i</a>
    <a href="http://sources.debian.net/prefix/j/">j</a>
    <a href="http://sources.debian.net/prefix/k/">k</a>
    <a href="http://sources.debian.net/prefix/l/">l</a>
    <a href="http://sources.debian.net/prefix/lib-/">lib-</a>
    <a href="http://sources.debian.net/prefix/lib3/">lib3</a>
    <a href="http://sources.debian.net/prefix/liba/">liba</a>
    <a href="http://sources.debian.net/prefix/libb/">libb</a>
    <a href="http://sources.debian.net/prefix/libc/">libc</a>
    <a href="http://sources.debian.net/prefix/libd/">libd</a>
    <a href="http://sources.debian.net/prefix/libe/">libe</a>
    <a href="http://sources.debian.net/prefix/libf/">libf</a>
    <a href="http://sources.debian.net/prefix/libg/">libg</a>
    <a href="http://sources.debian.net/prefix/libh/">libh</a>
    <a href="http://sources.debian.net/prefix/libi/">libi</a>
    <a href="http://sources.debian.net/prefix/libj/">libj</a>
    <a href="http://sources.debian.net/prefix/libk/">libk</a>
    <a href="http://sources.debian.net/prefix/libl/">libl</a>
    <a href="http://sources.debian.net/prefix/libm/">libm</a>
    <a href="http://sources.debian.net/prefix/libn/">libn</a>
    <a href="http://sources.debian.net/prefix/libo/">libo</a>
    <a href="http://sources.debian.net/prefix/libp/">libp</a>
    <a href="http://sources.debian.net/prefix/libq/">libq</a>
    <a href="http://sources.debian.net/prefix/libr/">libr</a>
    <a href="http://sources.debian.net/prefix/libs/">libs</a>
    <a href="http://sources.debian.net/prefix/libt/">libt</a>
    <a href="http://sources.debian.net/prefix/libu/">libu</a>
    <a href="http://sources.debian.net/prefix/libv/">libv</a>
    <a href="http://sources.debian.net/prefix/libw/">libw</a>
    <a href="http://sources.debian.net/prefix/libx/">libx</a>
    <a href="http://sources.debian.net/prefix/liby/">liby</a>
    <a href="http://sources.debian.net/prefix/libz/">libz</a>
    <a href="http://sources.debian.net/prefix/m/">m</a>
    <a href="http://sources.debian.net/prefix/n/">n</a>
    <a href="http://sources.debian.net/prefix/o/">o</a>
    <a href="http://sources.debian.net/prefix/p/">p</a>
    <a href="http://sources.debian.net/prefix/q/">q</a>
    <a href="http://sources.debian.net/prefix/r/">r</a>
    <a href="http://sources.debian.net/prefix/s/">s</a>
    <a href="http://sources.debian.net/prefix/t/">t</a>
    <a href="http://sources.debian.net/prefix/u/">u</a>
    <a href="http://sources.debian.net/prefix/v/">v</a>
    <a href="http://sources.debian.net/prefix/w/">w</a>
    <a href="http://sources.debian.net/prefix/x/">x</a>
    <a href="http://sources.debian.net/prefix/y/">y</a>
    <a href="http://sources.debian.net/prefix/z/">z</a>

  &ensp; | &ensp;
  Browse <a href="http://sources.debian.net/list/1/">by page</a>
</p>
<hr />
<div style="position: relative">
<div style="position: absolute; right: 0">
hosted by<br />
<a href="http://www.irill.org/">
  <img width="100px"
       style="vertical-align: middle;"
       src="../../../../../static/img/irill.png" alt="IRILL" />
</a>
</div>

<p>
  Debsources &mdash; Copyright (C) 2011&ndash;2014 Matthieu Caneill, Stefano
  Zacchiroli, and
  <a href="http://anonscm.debian.org/gitweb/?p=qa/debsources.git;a=blob;f=AUTHORS;hb=HEAD">contributors</a>.
  License:
  <a href="http://www.gnu.org/licenses/agpl.html">GNU AGPLv3</a>.
  <br />
  Hosted source files are available under their own
  <a href="http://www.debian.org/doc/debian-policy/ch-source.html#s-dpkgcopyright">copyright
  and licenses</a>.
  <br />
  Source code: <a href="http://anonscm.debian.org/gitweb/?p=qa/debsources.git">Git</a>.
  Contact: <a href="mailto:info@sources.debian.net">info@sources.debian.net</a>.
  Last update: Thu, 24 Jul 2014 04:15:25 -0000.
</p>
</div>
    </footer>
    
  </body>

<!-- Mirrored from sources.debian.net/src/dds/2.5.2+ddd105-1/debian/patches/sanitize-trydds.py by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 24 Jul 2014 06:08:03 GMT -->
</html>